lerobot/common/policies/sac/modeling_sac.py

```diff
@@ -0,0 +1,628 @@
+                    nn.LayerNorm(config.latent_dim),
+                    nn.Tanh(),
+                )
+            else:
+                self.image_enc_layers = nn.Sequential(
+                    nn.Conv2d(
+                        in_channels=config.input_shapes["observation.image"][0],
+                        out_channels=config.image_encoder_hidden_dim,
+                        kernel_size=7,
+                        stride=2,
+                    ),
+                    nn.ReLU(),
+                    nn.Conv2d(
+                        in_channels=config.image_encoder_hidden_dim,
+                        out_channels=config.image_encoder_hidden_dim,
+                        kernel_size=5,
+                        stride=2,
+                    ),
+                    nn.ReLU(),
+                    nn.Conv2d(
+                        in_channels=config.image_encoder_hidden_dim,
+                        out_channels=config.image_encoder_hidden_dim,
+                        kernel_size=3,
+                        stride=2,
+                    ),
+                    nn.ReLU(),
+                    nn.Conv2d(
+                        in_channels=config.image_encoder_hidden_dim,
+                        out_channels=config.image_encoder_hidden_dim,
+                        kernel_size=3,
+                        stride=2,
+                    ),
+                    nn.ReLU(),
+                )
+                dummy_batch = torch.zeros(1, *config.input_shapes["observation.image"])
+                with torch.inference_mode():
+                    self.image_enc_out_shape = self.image_enc_layers(dummy_batch).shape[1:]
+                self.image_enc_layers.extend(
+                    nn.Sequential(
+                        nn.Flatten(),
+                        nn.Linear(np.prod(self.image_enc_out_shape), config.latent_dim),
+                        nn.LayerNorm(config.latent_dim),
+                        nn.Tanh(),
+                    )
+                )
+            self.aggregation_size += config.latent_dim * self.camera_number
+        if "observation.state" in config.input_shapes:
+            self.state_enc_layers = nn.Sequential(
+                nn.Linear(
+                    in_features=config.input_shapes["observation.state"][0], out_features=config.latent_dim
+                ),
+                nn.LayerNorm(normalized_shape=config.latent_dim),
+                nn.Tanh(),
+            )
+            self.aggregation_size += config.latent_dim
+
+        if "observation.environment_state" in config.input_shapes:
+            self.env_state_enc_layers = nn.Sequential(
+                nn.Linear(
+                    in_features=config.input_shapes["observation.environment_state"][0],
+                    out_features=config.latent_dim,
+                ),
+                nn.LayerNorm(normalized_shape=config.latent_dim),
+                nn.Tanh(),
+            )
+        self.aggregation_size += config.latent_dim
```

Comment:
```suggestion
           self.aggregation_size += config.latent_dim
```
I think there's an indentation mistake here